Client/Frontend/Browser/Tab.swift

```diff
@@ -66,6 +66,9 @@ class Tab: NSObject {
 
   var secureContentState: TabSecureContentState = .unknown
 
+  var walletProvider: BraveWalletBraveWalletProvider?
```

Comment:
```suggestion
var walletProvider: BraveWalletProvider?
```

I think this wasn't meant to be listed twice

Reply:
It is correct, and is this way because Obj-C doesn't support namespacing and the [interface defined in the mojom file in brave-core includes "BraveWallet"](https://github.com/brave/brave-core/blob/8532627de764fdb25045255eb856c9c71f8e75c8/components/brave_wallet/common/brave_wallet.mojom#L125). In brave-core the generated C++ interface for this is `brave_wallet::mojom::BraveWalletProvider`. In Obj-C we namespace via name prefix BraveWallet (the mojom module name) to each generated type.  We try to rename this in Swift to `BraveWallet.{x}` but unfortunately protocols cannot be nested inside other types in Swift so we are forced to use the full prefixed name here

Reply:
Ahh verry interesting I'll take a deeper look into this for my own context, but I don't see how this could raise any additional concerns so I'd think this wouldn't need any other actions to be taken